### DISCUSSION TITLE
Network adapter

### DISCUSSION THREAD

**Discussion by Nirvana208** [BornToBeRoot/NETworkManager#2532] [Q&A]
We have a client who works remotely from home to her work computer. She reports that every time she attempts to gain remote access to her work computer, it crashes. After checking the logs, we observed that the network adapter is being removed and added every time she tries remote access. We've already tried updating Windows, disabling energy-saving mode for the network adapter, reinstalling the network adapter, disabling the antivirus, and running a malware scan, but the issue persists. Is there anything else we should check, and do you have a solution or suggestions to resolve this problem? Thanks

**Comment by BornToBeRoot** [comment]
Only questions/discussions about NETworkManager. No IT support here.
